jdl/jdl-importer.js

```diff
@@ -159,6 +159,7 @@ function checkForErrors(jdlObject, configuration) {
         let applicationType = configuration.applicationType;
         let databaseType = configuration.databaseType;
         let skippedUserManagement = configuration.skipUserManagement;
+        let blueprints = configuration.blueprints;
```

Comment:
You've forgotten to add the "blueprints" param to the jsdoc starting from l.63.

Reply:
added